### PR TITLE
Added infinite duration option for Alert

### PR DESCRIPTION
**Pull request by smuriana**
Congratulation for this work!
I will add this useful library to my project and I need this feature.
I added the infinity duration option and I added the example of this option in a new button in app sample.

### CODE REVIEW

File: alerter/src/main/java/com/tapadoo/alerter/Alert.java [hunk 1]
```diff
@@ -53,6 +53,7 @@
     private long duration = DISPLAY_TIME_IN_SECONDS;
 
     private boolean enableIconPulse = true;
+    private boolean enableInfiniteDuration = false;
```
Comment (kpmmmurphy) [inline alerter/src/main/java/com/tapadoo/alerter/Alert.java]: In Java, booleans have a default value of `false` - so you can remove the `= false;`.

File: alerter/src/main/java/com/tapadoo/alerter/Alert.java [hunk 2]
```diff
@@ -366,6 +369,15 @@ public void pulseIcon(final boolean shouldPulse) {
     }
 
     /**
+     * Set if the duration of the alert is infinite
+     *
+     * @param enableInfiniteDuration True if the duration of the alert is infinite
+     */
+    public void setEnableInfiniteDuration(boolean enableInfiniteDuration) {
```
Comment (kpmmmurphy) [inline alerter/src/main/java/com/tapadoo/alerter/Alert.java]: Mark boolean param as final.

**Comment by kpmmmurphy** [comment]
Looks good @smuriana - just update the version in the `build.gradle`, and in the README. Then it's good to merge!

**Comment by smuriana** [comment]
Ready! Thanks for the feedback ;)

**Pull request merged**